cclib/parser/gaussianparser.py

```diff
@@ -2212,5 +2212,29 @@ def extract_charges_spins(line,prop):
             except:
                 pass
 
+        # Extract Rotational Constants
+        # Example:
+        # Rotational constants (GHZ):           3.13081     1.24272     0.88960
+        # OR for linear molecules:
+        # Rotational constants (GHZ): ************ 12.73690 12.73690
+        # Note: rotational constant will be converted to wavenumber units (1/cm) to standardize across parsers
+        if line[:27] == ' Rotational constants (GHZ)':
+            if not hasattr(self, "rotcons"):
+                self.rotcons = []
+            splits = line.split()
+
+            # Determine if the molecule is linear and only has two constants
+            if '*' in line:  # linear molecule
+                self.rotcons.append([0.0]+[utils.float(splits[i])/29.9792458 for i in (-2, -1)])
```

Comment:
Could you use `utils.convertor` here see https://github.com/cclib/cclib/blob/07e7c8e185a684ed7d811f2e96810ff38b0489d0/cclib/parser/utils.py#L77

Reply:
Absolutely! I'll push a fixup commit with this shortly